examples/digital_fingerprinting/production/morpheus/dfp/stages/dfp_training.py

```diff
@@ -114,15 +148,23 @@ def on_data(self, message: MultiDFPMessage) -> MultiAEMessage:
         model.fit(train_df, epochs=self._epochs, val_data=validation_df, run_validation=run_validation)
         logger.debug("Training AE model for user: '%s'... Complete.", user_id)
 
-        output_message = MultiAEMessage(meta=message.meta,
-                                        mess_offset=message.mess_offset,
-                                        mess_count=message.mess_count,
-                                        model=model)
+        if (to_cm):
+            output_message = ControlMessage(message.meta)
+            output_message.set_metadata("user_id", user_id)
```

Comment:
What happens to the model here? Is it just lost?

Reply:
In this case, yes; we were not capturing the model in the ControlMessage paths. Will take a look.

Reply:
Serializing the model and storing it in the payload metadata in the next update.